docs/site/Examples.md

```diff
@@ -113,3 +113,8 @@ lb4 example hello-world
 Please follow the instructions in
 [Install LoopBack4 CLI](Getting-started.md#install-loopback-4-cli) if you don't
 have `lb4` installed yet.
+
+## Community examples
```

Comment:
Since you are here, let's add the following too:

```
The official examples are hosted at the following urls:

- https://github.com/strongloop/loopback4-example-shopping
- https://github.com/strongloop/loopback-next/tree/master/examples
```